compile.sh

```diff
@@ -37,7 +36,25 @@ if [ "$_has_dtbo" = "true" ]; then
 fi
 
 _self_dir=$(realpath $(dirname "$0"))
-. $_self_dir/compile_$_compiler.sh
+# TODO: Using source here so that all environment vars are inherited
```

Comment:
I think you can remove that alrady TODO comment :)

Reply:
/me Rubs eyes...
/me Rubs eyes again...
/me Knocks on the ISP...
/me Reboots the ISP...
That's better, guess I rushed the cleanup in. That's what review is for :)